app/src/main/java/com/example/trashminder/presentation/createdReminder/HomeFragment.kt

```diff
@@ -63,4 +87,72 @@ class HomeFragment : Fragment() {
             }
         }
     }
+
+    @RequiresApi(Build.VERSION_CODES.O)
+    @Composable
+    private fun CheckNotificationPermission(it: Reminder, context: Context) {
+
+        var hasNotificationPermission by remember {
```

Comment:
What does `remember` do?

Reply:
as the name says, it "remembers" the state of the variable during recompositions, in this case, the notification permission dialog will appear once, and whether the user allows or not, this state will remain throughout
 the rest of the app, unless there is a configuration change